sdk/search/azure-search-documents/azure/search/documents/_api_versions.py

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+_SUPPORTED_API_VERSIONS = [
+    "2019-05-06-Preview",
+]
+
+def validate_api_version(api_version):
+    # type: (str) -> None
+    """Raise error if api_version is invalid """
+    if not api_version:
+        return
+    if api_version not in _SUPPORTED_API_VERSIONS:
+        versions = '\n'.join(_SUPPORTED_API_VERSIONS)
+        raise ValueError("Unsupported API version '{}'. Please select from:\n{}".format(api_version, versions))
+
+
+def get_api_version(kwargs, default):
+    # type: (Dict[str, Any]) -> str
```

Comment:
is the type hint correct? there are 2 params above.

Reply:
Good catch! Fixed.